quickwit/quickwit-jaeger/src/lib.rs

```diff
@@ -63,8 +64,7 @@ mod metrics;
 // OpenTelemetry to Jaeger Transformation
 // <https://opentelemetry.io/docs/reference/specification/trace/sdk_exporters/jaeger/>
 
-/// A base64-encoded 16-byte array.
-type TraceId = String;
+type B64TraceId = [u8; 24];
```

Comment:
We have two types called B64TraceId.

One is a String the other is `[u8; 24]`;
It is a bit confusing. Can we rename one?